src/main/java/bookface/storage/JsonAdaptedBook.java

```diff
@@ -57,7 +75,20 @@ public Book toModelType() throws IllegalValueException {
         }
         final Author modelAuthor = new Author(author);
 
-        return new Book(modelTitle, modelAuthor);
+        if (isLoaned && returnDate == null) {
```

Comment:
Inconsequential question: should we put these `isLoaned() and returnDate == null` checks in Line 82 of JSONAdaptedPerson.java, because the JSONAdaptedPerson class is responsible for deserialising the loaned books?

Will it affect anything?

Reply:
I put the checks in JSONAdaptedBook because I wanted to keep the book fields as separate and less exposed to Person but I understand your point.

But if prof/peers point this out as a problem just throw the problem to me and i'll take the blame for it.

Reply:
Hmm, I don't think there are any bugs arising from this, so it's fine to leave it as it is